scripts/bert/pretraining_utils.py

```diff
@@ -241,13 +241,13 @@ def __init__(self, use_avg_len, num_ctxes, vocab):
         self._use_avg_len = use_avg_len
         self._num_ctxes = num_ctxes
         pad_val = vocab[vocab.padding_token]
-        self._batchify_fn = Tuple(Pad(pad_val=pad_val), # input_id
-                                  Pad(pad_val=pad_val), # masked_id
-                                  Pad(pad_val=0),       # masked_position
-                                  Pad(pad_val=0),       # masked_weight
-                                  Stack(),              # next_sentence_label
-                                  Pad(pad_val=0),       # segment_id
-                                  Stack())              # valid_length
+        self._batchify_fn = Tuple(Pad(pad_val=pad_val, round_to=8), # input_id
```

Comment:
Can it be controlled by an argument in command line?

Reply:
We should enable it by default, i don't think an argument is necessary

Reply:
From description
> TensorCore algorithms may only be invoked when the sequence length is a multiple of 8.

Reply:
Sorry I over looked that. So this script will never be executed on CPU or GPU without TensorCore?

Reply:
@TaoLv good point. You could open a PR to add a cmd flag. If we can't auto-detect when TensorCore is present, it may still be best to enable the flag by default but give users the option to disable it.